charts/consul/values.yaml

```diff
@@ -85,7 +85,7 @@ global:
   # image: "hashicorp/consul-enterprise:1.10.0-ent"
   # ```
   # @default: hashicorp/consul:<latest version>
-  image: "hashicorp/consul:1.10.3"
+  image: "hashicorp/consul:1.11.0"
```

Comment:
This should still be consul 1.10.3
Consul 1.11.0 will only be available as a beta and users will have to explicitly update the values file to use the beta2 image.